integration_tests/image_test.go

```diff
@@ -90,3 +91,19 @@ func pineconeContainer(ctx context.Context, t *testing.T) testcontainers.Contain
 
 	return container
 }
+
+func redisContainer(ctx context.Context, t *testing.T) testcontainers.Container {
+	req := testcontainers.ContainerRequest{
+		Image:        "redis/redis-stack:latest",
```

Comment:
pin the redis version - otherwise the test might fail if redis behaviour changes between versions.

Reply:
Done in https://github.com/qdrant/migration/pull/44/commits/7ce29f6049fa55317d5eca6900311d265d625f5d.